Makefile

```diff
@@ -1,8 +1,39 @@
 PROJ_VERSION = 6.3.0
-
-install_proj:
+install_proj: ## Install PROJ and clang (requirements to use proj crate)
 	sudo apt update
-	sudo apt install -y wget build-essential pkg-config sqlite3 libsqlite3-dev clang
-	wget https://github.com/OSGeo/proj.4/releases/download/$(PROJ_VERSION)/proj-$(PROJ_VERSION).tar.gz && tar -xzvf proj-$(PROJ_VERSION).tar.gz
-	cd proj-$(PROJ_VERSION) && ./configure --prefix=/usr && make && sudo make install
+	sudo apt install -y clang
+
+	# Needed only for proj install
+	sudo apt install -y wget build-essential pkg-config sqlite3 libsqlite3-dev
+
+	# remove PROJ system version from packages
+	sudo apt remove libproj-dev
+
+	wget https://github.com/OSGeo/proj.4/releases/download/$(PROJ_VERSION)/proj-$(PROJ_VERSION).tar.gz
+	tar -xzvf proj-$(PROJ_VERSION).tar.gz
+	pushd proj-$(PROJ_VERSION)
+	./configure --prefix=/usr && make
+	sudo make install
+	popd
 	rm -rf proj-$(PROJ_VERSION) proj-$(PROJ_VERSION).tar.gz
```

Comment:
minor: do we want to `apt remove` once Proj has been compiled and installed?

Reply:
It was a comment previously (before the makefile even appeared), as we do not want to side-effect in removing some packages of the user.
So it would be only for doc if we keep it that way, and I considered commenting/separating the packages needed for proj install was covering for that (above).